doc/doxygen/src/css/riot.css

```diff
@@ -79,7 +79,7 @@ dl {
   color: #555555;
 }
 #MSearchSelectWindow {
-  background-color: #000000;
+  background-color: #000;
```

Comment:
??!?

Reply:
Arf, `make static-test` always generates a diff in `riot.css`, I don't know why.

I removed the diff.

Reply:
Maybe you have some different version of lessc or lesscpy installed?

Reply:
I have this:
```
$ lessc --version
lessc 2.7.2 (Less Compiler) [JavaScript]
```